src/content/docs/dropins/checkout/containers/payment-methods.mdx

```diff
@@ -113,39 +115,38 @@ CheckoutProvider.render(PaymentMethods, {
 })($paymentMethods),
 ```
 
-## Example 3: How to customize a payment method using the `slots.Handlers` configuration
+## Example 3: How to customize a payment method using the `slots.Methods` configuration
 
-The following example renders the `PaymentMethods` container on a checkout page, displaying the available payment methods in the element with the class `checkout__payment-methods`, providing a specific handler for `PaymentServices` payment method:
+The following example renders the `PaymentMethods` container on a checkout page, displaying the available payment methods in the element with the class `checkout__payment-methods`, providing a specific handler for Payment Services' `CREDIT_CARD` payment method:
```

Comment:
```suggestion
The following example renders the `PaymentMethods` container on a checkout page, displaying the available payment methods in the element with the class `checkout__payment-methods`, providing a specific handler for the Payment Services `CreditCard` payment method:
```

Reply:
we are using `CreditCard` in the rest of our docs

Reply:
Yes, but we are providing a handler for Payment Service' `CREDIT_CARD` payment method as specified in the code example:

```
[PaymentMethodCode.CREDIT_CARD]: { ... }
```

Also, the apostrophe after Payment Service is intended here, as it is the `CREDIT_CARD` payment method of Payment Services, i.e., Payment Service' `CREDIT_CARD` payment method. It is a plural possessive: https://www.hamilton.edu/academics/centers/writing/seven-sins-of-writing/4#:~:text=Plural%20Possessive,do%20end%20in%20%E2%80%9Cs.%E2%80%9D